ch6/Homework_Problems.ipynb

```diff
@@ -8,23 +8,69 @@
     "\n",
     "Write a class named Fraction that represents a rational number (a number that can be expressed as the quotient of two integers). It is required to implement the following methods:\n",
     "\n",
-    "The __init__(self, numerator, denominator) method should accept integer values for the numerator and denominator arguments and set instance attributes of the same name. If the denominator is 0, raise a ZeroDivisionError exception. Use the [math.gcd function](https://docs.python.org/3/library/math.html#math.gcd) to find the greatest common divisor (GCD) of the numerator and denominator and then divide each of them by it to \"normalize\" the fraction. For example, the fraction 28/20 will get normalized to 7/5 since the GCD of 28 and 20 is 4:\n",
-    "\n",
+    "The `__init__(self, numerator, denominator)` method should accept integer values for the numerator and denominator arguments and set instance attributes of the same name. If the denominator is 0, raise a `ZeroDivisionError` exception. Use the [math.gcd function](https://docs.python.org/3/library/math.html#math.gcd) to find the greatest common divisor (GCD) of the numerator and denominator and then divide each of them by it to \"normalize\" the fraction. For example, the fraction 28/20 will get normalized to 7/5 since the GCD of 28 and 20 is 4:\n",
+    "```\n",
     ">>> x = Fraction(28, 20)\n",
     ">>> x\n",
     "Fraction(7, 5)\n",
+    "```\n",
     "Implement the basic binary operators (+, -, *, /) so that a Fraction can combined with either another fraction or an integer. All methods should return a new Fraction. Note that you may need to implement \"reversed\" operators for arithmetic with integers to fully work.\n",
     "\n",
-    "- The __neg__ method should return a new Fraction instance with the numerator negated.\n",
-    "- The __repr__ method should return a string of the form Fraction(a, b) where a and b are the numerator and denominator, respectively.\n",
+    "- The `__neg__` method should return a new Fraction instance with the numerator negated.\n",
+    "- The `__repr__` method should return a string of the form Fraction(a, b) where a and b are the numerator and denominator, respectively.\n",
     "\n",
     "Addition should also work:\n",
     "\n",
     "```\n",
-    "__repr__(Frac(1, 2) + Frac(1, 3)) = Fraction(5/6)\n",
+    "__repr__(Fraction(1, 2) + Fraction(1, 3)) = Fraction(5/6)\n",
     "```"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The below should raise a `ZeroDivisionError`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from problem_1 import Fraction\n",
```

Comment:
Please put the file and the class (with a `pass`) in the directory so they're not running into an import error. See ch2 for examples of this.

<img width="734" alt="Screen Shot 2021-01-08 at 8 59 19 AM" src="https://user-images.githubusercontent.com/5744164/104030013-58176200-5190-11eb-9fba-75fb76769193.png">